spec/requests/api/v1/topics/view_spec.rb

```diff
@@ -0,0 +1,28 @@
+## Get all topics
```

Comment:
Rename this file to match with the controller method you are testing.. 
`spec/requests/api/v1/topics/view_spec.rb` --> `spec/requests/api/v1/topics/index_spec.rb`